endpointmanager/cmd/CHPLpopulator/main.go

```diff
@@ -257,13 +256,14 @@ func containsEndpoint(endpointEntryList []endpointEntry, url string) bool {
 	return false
 }
 
-func containsFileName(endpointEntryList []endpointEntry, filename string) bool {
+func containsFileName(endpointEntryList []endpointEntry, filename string) int {
+	matchedFiles := 0
 	for _, e := range endpointEntryList {
 		if e.FileName == filename+"EndpointSources.json" {
```

Comment:
I don't think this works, when i work through this scenario:
1st athenahealth list: there are no list named `athenahealthEndpointSources.json`, so the file gets called `athenahealthEndpointSources.json`
2nd athenahealth list: there is a file named `athenahealthEndpointSources.json`, so matchedFiles is increased (and then increased again on line 187?) and the new file is called `athenahealth2EndpointSources.json`
3rd athenahealth list: there is still just one file named `athenahealthEndpointSources.json`, so the same thing happens and the new file is again called `athenahealth2EndpointSources.json` again.

I think maybe it could make more sense to put the index at the end (`athenahealthEndpointSources1`, etc.), and then on this line check if `e.FileName` contains "athenahealthEndpointSources", which would catch all of the lists? Does that make sense?

Reply:
Honestly we might have to rethink how we name our files in general, since one of the files for an endpoint list that was just added is `com/FHIR_Service_URL_08162022EndpointSources.json` which I don't think we'd want moving forward....